hedera-node/src/main/java/com/hedera/services/state/logic/BlockManager.java

```diff
@@ -47,6 +48,7 @@ public class BlockManager {
     private static final Logger log = LogManager.getLogger(BlockManager.class);
 
     private final long blockPeriodMs;
+    private final boolean logEveryTransaction;
```

Comment:
I put it there because I saw that `bootstrapProperties` was already being injected into `BlockManager` and the property that was being used was `HEDERA_RECORD_STREAM_LOG_PERIOD` which already seemed to be coupling the `RecordStreaming` with the `BlockManager`.

Should I inject `bootstrapProperties` into  `RecordStreaming` as well and move it there?

Reply:
Ah got it, in general want a class to use of all its members, but this is a very minor violation so either way is fine. 😁